lesson_02/fib_sequence/fib_sequence.go

```diff
@@ -0,0 +1,11 @@
+package fib_sequence
+
+import "fmt"
+
+func FibNums(n int) {
+	var a, b int = 1, 1
+	for i := 0; i < n; i++ {
+		fmt.Println(a)
```

Comment:
This is not a good idea to print inside function. Function should return value, and we can save this value, print or do something else than.